package.json

```diff
@@ -11,7 +11,8 @@
     "aos:load": "node tools/bundle-aos.mjs && node tools/load-aos.mjs",
     "aos:spawn": "node tools/spawn-aos.mjs",
     "test": "yarn aos:build && node --test --test-concurrency 1 --experimental-wasm-memory64 **/*.test.mjs",
-    "prepare": "husky"
+    "prepare": "husky",
+    "install-lua-deps": "sh tools/install-lua-deps.sh"
```

Comment:
i don't see this file anywhere

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/32/commits/34316ae44c426e93e4a7addd48652025a0750696

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/32/commits/81da676a2e3ac2bbdd7bb9e24fc774aae8c39e6b